content/blog/2020-06-08-june-20-heartbeat.md

```diff
@@ -0,0 +1,213 @@
+---
+title: June '20 Heartbeat
+date: 2020-06-08
+description: |
+  Catch up on new DVC releases, talks, and projects in our community. 
+  This month, learn about finishing touches on DVC 1.0, DVC in biomedical
+  research, recommended reading and upcoming MLOps talks.
+descriptionLong: |
+  Catch up on new DVC releases, talks, and projects in our community. 
+  This month, learn about finishing touches on DVC 1.0, DVC in biomedical
+  research, recommended reading and upcoming MLOps talks.
+picture: 2020-05-14/May_20_Heartbeat.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/may-20-community-gems/398
+tags:
+  - Heartbeat
+  - Udemy course
+  - DVC 1.0
+  - Pipelines
+  - Plots
+  - MLOps
+---
+
+Welcome to the June Heartbeat, our [monthly roundup of cool happenings](#news),
```

Comment:
probably "news" link is not needed? and probably we can add link to the "other noteworthy" to the "coming up soon" section?

Reply:
or even mention that section explicitly among others